packages/client/manifest.json

```diff
@@ -3157,6 +3157,57 @@
       }
     ]
   },
+  "codescanner": {
+    "name": "Barcode/QR Scanner",
+    "icon": "Camera",
+    "styles": [
+      "size"
+    ],
+    "draggable": true,
+    "illegalChildren": [
+      "section"
+    ],
+    "settings": [
+      {
+        "type": "field/barcodeqr",
+        "label": "Field",
+        "key": "field",
+        "required": true
+      },
+      {
+        "type": "text",
+        "label": "Label",
+        "key": "label"
+      },
+      {
+        "type": "text",
+        "label": "Button",
```

Comment:
NAB: let's rename this to "Button text" to make it more obvious that's what it is